spring-boot-cxf-jaxws-xml/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.apache.camel.springboot.example</groupId>
+    <artifactId>spring-boot-cxf-jaxws-xml</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
For most of the examples we have we have 4.10.3-SNAPSHOT as version - would that make sense here too?